src/openms_gui/source/VISUAL/APPLICATIONS/TOPPViewBase.cpp

```diff
@@ -1534,6 +1547,10 @@ namespace OpenMS
   {
     if (!ws_.currentSubWindow())
     {
+      if (activeSubwindowID_ < (Size) ws_.subWindowList().size())
```

Comment:
when does this case happen? this looks a bit fishy

Reply:
hopefully always otherwise the indexing fails in the next line.
That is why we return nullptr if not.
Can you elaborate?